docs/user-guide/install.md

```diff
@@ -1,16 +1,13 @@
-# Introduction
-
-vSphere Docker Volume Service is simple to install. It has zero configuration and and zero credential management post install.
-
-[Tagged releases](https://github.com/vmware/docker-volume-vsphere/releases) include the software bundles to install on ESX and on the VM.
-
-In addition the ```make build-all``` will generate the packages.
+###...
+# Installation
 
+The installation has two parts – installation of the backend on ESX and installation of Docker plugin on the hosts where you plan to run containers with storage needs. You can use the [Tagged releases](https://github.com/vmware/docker-volume-vsphere/releases) in Github to get latest version or pick up a specific version. 
```

Comment:
Can we please remove this line - "You can use the [Tagged releases](https://github.com/vmware/docker-volume-vsphere/releases) in Github to get latest version or pick up a specific version"